src/components/Search.js

```diff
@@ -22,23 +21,41 @@ class Search extends React.Component {
     }
   }
 
+  onChange = e => {
+    e.preventDefault();
+    const newText = get(e, "nativeEvent.text", "");
+    const { searchTextInputChanged } = this.props;
+    searchTextInputChanged(newText);
+  };
+
   render() {
-    const { view, _searchTextInputChanged, load_user, user_tl } = this.props;
+    const {
+      load_user,
+      screen,
+      searchedText,
+      searchFocus,
+      user_tl
+    } = this.props;
+
     return (
       <Container>
         <Container>
           <Header searchBar rounded>
             <Item>
-              <Icon onPress={() => load_user()} name="ios-search" />
+              <Icon onPress={load_user} name="ios-search" />
               <Input
                 placeholder="Rechercher"
-                onChangeText={text => _searchTextInputChanged(text)}
+                value={searchedText}
+                onChange={this.onChange}
+                autoFocus={searchFocus}
```

Comment:
I used them during the debugging phase. We might not need `autoFocus` and we might be able to replace `onChange` with the original `onChangeText`, I'll see on Saturday/Sunday